src/jit/morph.cpp

```diff
@@ -2995,8 +3000,6 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
         {
```

Comment:
@briansull it looks like this comment is not attached to the right line. Where should it be?